docs/content/2.deploy/20.providers/zeabur.md

```diff
@@ -0,0 +1,18 @@
+# Zeabur
+
+Deploy Nitro apps to [Zeabur](https://zeabur.com).
+
+**Preset:** `zeabur` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+::alert
+**Zero Config Provider**
+:br
+Integration with this provider is possible with zero configuration. ([Learn More](/deploy/#zero-config-providers))
+::
+
+## Deploy using Git
+
+1. Push your code to your git repository (Currently only GitHub supported).
+2. [Import your project](https://zeabur.com/docs/get-started) into Zeabur.
+3. Zeabur will detect that you are using Nitro and will enable the correct settings for your deployment.
```

Comment:
How is it possible today?

Reply:
Zeabur has an open-sourced project [zbpack](https://github.com/zeabur/zbpack) to detect the project configuration, including programing language, framework and other configs.

Currently, Zeabur will detect a Nuxt.js project and auto inject the `NITRO_PRESET` environment variable to build the project with proper command (https://github.com/zeabur/zbpack/blob/main/pkg/zeaburpack/main.go#L213-L221). But I think we need a stable method to determine a project is build with Nitro (even if it's not Nuxt.js project).

Reply:
@pi0 I suddenly realized something: after this Pull Request is merged, all frameworks developed based on Nitro (whether they're Nuxt.js or not) will automatically use `NITRO_PRESET=zeabur` when building, determined by the `ZEABUR=1` environment variable. The output from the completed build will then comply with the Zeabur format, eliminating the need for additional conversions through zbpack as required by current Nuxt.js projects!